src/components/DateTimePicker/DateTimePicker.jsx

```diff
@@ -124,6 +124,8 @@ const propTypes = {
   disabled: PropTypes.bool,
   /** show the relative custom range picker */
   showRelativeOption: PropTypes.bool,
+  /** get the tooltip value for range selections */
+  getTooltipValue: PropTypes.func,
```

Comment:
I think it makes more sense if we rename this to `renderPresetTooltipText` to mirror the prop that it ultimately gets used for, and follow the same naming convention that we see for props that render things (function or otherwise), like `renderIcon`.  The comment/documentation can specify the purpose, parameter, and that it should return a string.

```suggestion
  /**
   * Function hook used to provide the appropriate tooltip content for the preset time
   * picker. This function takes in the currentValue and should return a string message.
   */
  renderPresetTooltipText: PropTypes.func,
```

Reply:
I had named it initially as only for preset tooltip but thought a more generic prop name will leave the option open to set user defined tooltips for other types as well. However, you are correct. The user may expect the custom tooltip to show for other selection types which the component is not doing thereby resulting in a defect.